core/src/main/java/com/sequenceiq/cloudbreak/service/credential/CredentialParameterSetterUtil.java

```diff
@@ -37,7 +37,8 @@ public void setProperParameters(String cloudPlatform, CredentialV4Base response,
         } else if (CloudPlatform.CUMULUS_YARN.name().equals(cloudPlatform)) {
             response.setCumulus(exec(() -> new Json(parameters).get(CumulusYarnCredentialV4Parameters.class), CumulusYarnCredentialV4Parameters.class));
         } else if (CloudPlatform.AZURE.name().equals(cloudPlatform)) {
-            response.setAzure(exec(() -> new Json(parameters).get(AzureCredentialV4Parameters.class), AzureCredentialV4Parameters.class));
+            response.setAzure(conversionService.convert(parameters, AzureCredentialV4ResponseParameters.class));
+            //response.setAzure(exec(() -> new Json(parameters).get(AzureCredentialV4ResponseParameters.class), AzureCredentialV4ResponseParameters.class));
```

Comment:
Commented line should be removed

Reply:
it is not a new comment but makes the test more understandable (reflection stuffs can be messy). the class comes from a renamed test from the past